src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Utilities/SequencialTaskExecutor.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Utilities
```

Comment:
We're moving to these the appropriate namespaces/folders, in this case Microsoft.VisualStudio.Threading.Tasks.